.github/workflows/cron-nightly-rust.yml

```diff
@@ -0,0 +1,57 @@
+name: Update nightly Rust
+on:
+  schedule:
+    - cron: "29 17 1 * *" # At 17:29 on day-of-month 1.
```

Comment:
Why this specific time?

Reply:
Ohkayy. So that's the Ramanujan number from the PR description. I guess it's fine. 🤷

Reply:
Yes also imagine how many stuff around the world runs at 0h UTC.